core/math/random_pcg.cpp

```diff
@@ -32,10 +32,10 @@
 
 #include "core/os/os.h"
 
-RandomPCG::RandomPCG(uint64_t seed, uint64_t inc) :
+RandomPCG::RandomPCG(uint64_t p_seed, uint64_t inc) :
```

Comment:
If you change the parameter name here, you should change it in the header file too. Might be worth to take the opportunity to prefix all parameters in RandomPCG functions with `p_`, as is normally the convention in Godot's codebase.

Reply:
Done.